mp/src/game/client/momentum/ui/leaderboards/LeaderboardsTimes.cpp

```diff
@@ -325,9 +335,17 @@ void CLeaderboardsTimes::SetPlaceColors(SectionedListPanel* panel, TimeType_t ty
         if (itemCount > 1)
         {
             panel->SetItemBgColor(panel->GetItemIDFromRow(1), m_cSecondPlace);
-
             if (itemCount > 2)
+            {
                 panel->SetItemBgColor(panel->GetItemIDFromRow(2), m_cThirdPlace);
+                if(itemCount > 3)
```

Comment:
Also minor, but we usually space this out. Eg. (`if ()`, `for ()`).

Reply:
I should probably point CLion towards the clangformat file but I'll also do another style pass including OVERRIDE etc.